translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionIterableAssertion.kt

```diff
@@ -31,6 +31,7 @@ enum class DescriptionIterableAssertion(override val value: String) : StringBase
     WARNING_ADDITIONAL_ENTRIES("zusätzliche Einträge entdeckt"),
     WARNING_MISMATCHES("folgende Einträge erfüllten keine Aussage (Diskrepanzen)"),
     WARNING_MISMATCHES_ADDITIONAL_ENTRIES("Diskrepanzen und zusätzliche Einträge entdeckt"),
+    NEXT("next"),
```

Comment:
Please change it to `NEXT_ELEMENT("hat ein nächstes Element")`

Reply:
What about representation field(https://github.com/robstoll/atrium/blob/master/core/api/atrium-core-api-common/src/main/kotlin/ch/tutteli/atrium/assertions/builders/AssertionBuilder.kt#L208) in this case? Could it be null?
I did not fully get the idea of this DescriptiveAssertion.

Reply:
A descriptive assertion gets formatted as follows:
```
* description: representation
```

Reply:
Yep, I got that part, but if description is `a next element` what's gonna be in representation?

Reply:
Originally I thought it's gonna be like
Description - HAS
Representation - NEXT
But not sure now :)

Reply:
in your case description is `has` and representation is `a next element`. In most cases the representation is an object or such but here it is only a description of a property. The way you did it first with `RawString.create(...)` was the way to go